package.json

```
@@ -15,15 +15,13 @@
     "build:dev": "webpack --mode=development && npm run build:es",
     "docs:examples": "node tooling/docs/examples-to-md.js examples/node/*.js",
     "docs:api": "node apidoc.js",
-    "docs:changelog": "cp CHANGELOG.md docs/CHANGELOG.md",
-    "lint": "commitlint --from HEAD~50 && eslint .",
+    "lint": "eslint .",
     "test": "nyc mocha './test/unit/' './test/integration/'",
-    "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
     "test:integration": "nyc mocha './test/integration/'",
     "test:unit": "nyc mocha './test/unit/'",
     "test:watch": "mocha './test/unit/' './test/integration/' --watch",
     "prepare": "npm run build",
-    "prepublishOnly": "npm run docs:examples && npm run docs:changelog",
+    "prepublishOnly": "npm run docs:examples",
     "release": "standard-version --skip.tag"
```

```suggestion
    "release": "standard-version --skip.tag --infile docs/CHANGELOG.md"
```

Configure release output to new location